File: Back_end/Model/Rating.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    public class Rating :Basedb
    {
        public Guid Id_Order { get; set; }
        public Guid Id_product { get; set; }
        public string Id_customer { get; set; }
        public int Evaluate {  get; set; }
        public string Comment { get; set; }
        public int Status { get; set; }



    }
}

File: DoAn5.UI/src/app/model/rating.ts
import { BasedbDto } from "./Common/BaseDto";

export interface RatingDto  {
    id_Order: string;
    id_product: string;
    id_customer: string;
    evaluate: number;
    comment: string;
    status: number;
}

export interface GetRatingDto extends BasedbDto {
    id_Order: string;
    id_product: string;
    id_customer: string;
    evaluate: number;
    comment: string;
    status: number;
    username: string
}
export interface CreateRatingDto {
    listRating: RatingDto[];
}
